src/containers/ClauseTemplate/ClauseLogicEditor.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Tab } from 'semantic-ui-react';
+import ErgoEditor from '../BaseEditors/ErgoEditor';
+import { editClauseLogicAction } from '../../actions/clauseTemplatesActions';
+
+
+/**
+ * A code editing component for logic files
+ * @param {*} props the properties for the component
```

Comment:
We should probably include this documentation on all of the editor files (`ClauseGrammarEditor`, `ClauseMetadataEditor`, and `ClauseExampleTextEditor`)